src/app/views/user/user-settings/user-profile/user-profile.component.html

```diff
@@ -1,82 +1,122 @@
+           [getData]="getLanguages()">
+        </autocomplete-dropdown>
 
-      <label for="education">{{'profile.form.education' | translate | uppercase}}</label>
-      <textarea name="education" id="education" class="material-input" [disabled]="!editMode" [(ngModel)]="user.education"></textarea>
-    </div>
+        <div class="profile-input-languages-table-container">
+          <div class="profile-input-languages-table-row-container" *ngFor="let userLanguage of user.userLanguages">
+            <div class="profile-input-languages-table-row-inner-container" *ngIf="userLanguage.proficiency.proficiency !== 5">
+              <div class="profile-input-languages-table-row-inner-language-container"><p>{{userLanguage.language.name}}</p></div>
+              <div class="profile-input-languages-table-row-inner-icon-container" (click)="onRemoveUserLanguage(userLanguage)">
+                <i class="fa fa-times" aria-hidden="true"></i>
+              </div>
+              <div class="profile-input-languages-table-row-inner-dropdown-container">
+                <autocomplete-dropdown [lookupType]="'list'" [enumList]="languageProficiencyLevelsAvailable" [(destination)]="userLanguage.proficiency" [placeholder]="'language.proficiency.placeholder' | translate" [autocompleteResultLabelFunction]="namePropertyLabel"
+                    [selectedItemLabelFunction]="namePropertyLabel">
+                </autocomplete-dropdown>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
 
-    <div class="material-group">
-      <i class="icon-ic-tool"></i>
+      <div class="profile-input-container">
+        <label>{{'user.profile.form.skills.label' | translate}}:</label>
+        <textarea class="form-control material-input" name="skills" id="skills" class="form-control material-input" autosize type="text" placeholder="{{'user.profile.form.skills.placeholder' | translate}}" [(ngModel)]="user.skills"></textarea>
+      </div>
 
-      <label for="skills">{{'profile.form.skills' | translate | uppercase}}</label>
-      <textarea name="skills" id="skills" class="material-input" [disabled]="!editMode" [(ngModel)]="user.skills"></textarea>
-    </div>
+      <div class="profile-input-container">
+        <label>{{'user.profile.form.work.experience.label' | translate}}:</label>
+        <textarea class="form-control material-input" name="workExperience" id="work-experience" class="form-control material-input" autosize type="text" placeholder="{{'user.profile.form.work.experience.placeholder' | translate}}" [(ngModel)]="user.workExperience"></textarea>
+      </div>
+<!--
+Documents not supported yet. TODO implement when documents supported in backend.
```

Comment:
Lets create a task for this in Asana so we don't forget :) (I'll try to remember to do it later, can't do it from this computer...)